src/scss/_main.scss

```diff
@@ -12,22 +12,26 @@
 /// @param {String} $layout-name - Name of the layout (e.g. S)
 /// @param {Number} $layout-width - Layout width in px
 /// @param {Number} $gutter-width [null] - Gutter width in px
-@mixin oGridAddLayout($layout-name, $layout-width, $gutter-width: null) {
+/// @param {Number} $outer-margin-width [null] - Outer margin width in px
+@mixin oGridAddLayout($layout-name, $layout-width, $gutter-width: null, $outer-margin-width: null) {
 	$temp-layouts: ();
 	$temp-gutters: (default: #{oGridGutter()});
+	$temp-outer-margins: (default: #{oGridGutter()});
 
 	// Add the new layout in the correct position:
-	// (we want $o-grid-layouts and $o-grid-gutters to ordered from the smallest to the largest layout)
+	// (we want $o-grid-layouts and $o-grid-gutters and $o-grid-outer-margins to be ordered from the smallest to the largest layout)
 	@for $index from 1 through length($o-grid-layouts) {
```

Comment:
No use case currently, but if it was the opposite, we probably would be saying the same about it making it not overrideable, but it takes a bit more effort to not make it overrideable. I was just wondering if there was a reason